libs/block-template-react/package.json

```diff
@@ -59,7 +59,7 @@
     "displayName": "Template",
     "examples": [
       {
-        "name": "World"
+        "https://blockprotocol.org/@blockprotocol/types/property-type/name/": "World"
```

Comment:
I'm assuming it's not an issue that `https://blockprotocol.org/@blockprotocol/types/property-type/name/` is not a real type, and that we plan on publishing common types like this when we have a type nexus

Reply:
https://blockprotocol.org/@blockprotocol/types/property-type/name/v/1 should actually exist - it's a property on https://blockprotocol.org/@blockprotocol/types/entity-type/thing/v/2